interfaces/repo.go

```diff
@@ -249,7 +249,7 @@ func (r *Repository) AddPlug(plug *snap.PlugInfo) error {
 	r.m.Lock()
 	defer r.m.Unlock()
 
-	snapName := plug.Snap.Name()
```

Comment:
Changes to the repository are fine, we don't use `_` as a special delimiter of any kind.